packages/components/app/styles/components/application-state.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APPLICATION-STATE COMPONENT
+//
+$hds-application-state-padding: 12px 0;
+
+.hds-application-state {
+  align-self: center;
+  width: 20rem;
```

Comment:
I wouldn't mind either way; since it's only used once, it seemed a little heavy-handed to tokenize. WDYT?

Reply:
It seems like we like to expose values like this as tokens in case consumers want to customize to make it easier for them to? That's the only reason I would make it a token but not sure if we actually need to.